### PR TITLE
Honor product/company name cases - nitpicking (part 6) 

### PR DESCRIPTION
**Pull request by OnkarRuikar**
## Summary
Continuing #14400 

Corrected following names:
- CSS
- API
- Windows
- Mac
- WebKit
- Android
- Linux
- Node.js

Note: Suggest more names to correct like XML, JSON,  the Internet vs internet etc.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/glossary/distributed_denial_of_service/index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ The United States Computer Emergency Readiness Team (US-CERT) defines symptoms o
 - Inability to access any website
 - Dramatic increase in the number of spam emails received—(this type of DoS attack is considered an email bomb)
 - Disconnection of a wireless or wired internet connection
-- Longterm denial of access to the Web or any internet services
+- Longterm denial of access to the Web or any Internet services
```
Comment (OnkarRuikar) [inline files/en-us/glossary/distributed_denial_of_service/index.md:29]:
```suggestion
- Longterm denial of access to the Web or any internet services
```

File: files/en-us/learn/getting_started_with_the_web/dealing_with_files/index.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ For now, this is about all you need to know.
 
 That is about it for now. Your folder structure should look something like this:
 
-![A file structure in mac os x finder, showing an images folder with an image in, empty scripts and styles folders, and an index.html file](file-structure.png)
+![A file structure in Mac os x finder, showing an images folder with an image in, empty scripts and styles folders, and an index.html file](file-structure.png)
```
Comment (teoli2003) [inline files/en-us/learn/getting_started_with_the_web/dealing_with_files/index.md:86]:
```suggestion
![A file structure in macOS finder, showing an images folder with an image in, empty scripts and styles folders, and an index.html file](file-structure.png)
```

File: files/en-us/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ The next images show the command prompts available in Windows — there's a good
 
 And below, you can see the macOS terminal application.
 
-![A basic vanilla mac terminal](mac-terminal.png)
+![A basic vanilla Mac terminal](mac-terminal.png)
```
Comment (teoli2003) [inline files/en-us/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md:70]:
```suggestion
![A basic vanilla macOS terminal](mac-terminal.png)
```

File: files/en-us/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md [hunk 4]
```diff
@@ -199,7 +199,7 @@ In the case of `ls`, the `-l` (_dash ell_) option gives you a listing with one f
 
 Below is a screenshot with a "vanilla" macOS terminal at the top, and a customized terminal with some extra icons and colors to keep it looking lively — both showing the results of running `ls -l`:
 
-![A vanilla mac terminal and a more colorful custom mac terminal, showing a file listing - the result of running the ls -l command](mac-terminals-ls.png)
+![A vanilla Mac terminal and a more colorful custom Mac terminal, showing a file listing - the result of running the ls -l command](mac-terminals-ls.png)
```
Comment (teoli2003) [inline files/en-us/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md:202]:
```suggestion
![A vanilla macOS terminal and a more colorful custom macOS terminal, showing a file listing - the result of running the ls -l command](mac-terminals-ls.png)
```

File: files/en-us/web/api/barcode_detection_api/index.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ The Barcode Detection API detects linear and two-dimensional barcodes in images.
 
 ## Concepts and usage
 
-Support for barcode recognition within web apps unlocks a variety of use cases through supported barcode formats. QR codes can be used for online payments, web navigation or establishing social media connections, aztec codes can be used to scan boarding passes and shopping apps can use EAN or UPC barcodes to compare prices of physical items.
+Support for barcode recognition within web apps unlocks a variety of use cases through supported barcode formats. QR codes can be used for online payments, web navigation or establishing social media connections, Aztec Codes can be used to scan boarding passes and shopping apps can use EAN or UPC barcodes to compare prices of physical items.
```
Comment (teoli2003) [inline files/en-us/web/api/barcode_detection_api/index.md:18]:
```suggestion
Support for barcode recognition within web apps unlocks a variety of use cases through supported barcode formats. QR codes can be used for online payments, web navigation or establishing social media connections, Aztec codes can be used to scan boarding passes and shopping apps can use EAN or UPC barcodes to compare prices of physical items.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
Thanks a lot!

**Pull request merged**